.travis.yml

```diff
@@ -1,30 +1,45 @@
 language: php
 
-php:
-  - 5.3
-  - 5.4
-  - 5.5
-  - 5.6
-  - hhvm
+sudo: false
 
-env:
-  - SYMFONY_VERSION=2.1.*
-  - SYMFONY_VERSION=2.2.*
-  - SYMFONY_VERSION=2.3.*
-  - SYMFONY_VERSION=dev-master
+branches:
+  only:
+    - master
+
+matrix:
+  include:
+    - php: 5.3
+      env:
+        - deps=low
+        - SYMFONY_VERSION=2.7.*
+    - php: 5.4
+      env:
+        - deps=high
+        - SYMFONY_VERSION=2.7.*
+    - php: 5.5
+      env:
+        - deps=low
+        - SYMFONY_VERSION=2.8.*
+    - php: 5.6
+      env:
+        - deps=high
+        - SYMFONY_VERSION=3.0.*
+    - php: 7.0
+      env:
+        - deps=low
+        - SYMFONY_VERSION=3.1.*
+    - php: 7.1
+      env:
+        - deps=high
+        - SYMFONY_VERSION=3.1.*
```

Comment:
The PHP BC policy is quite messy and not really followed (see 7.1 BC breaks, and proposals for 7.2). I want to test every PHP version, I think it's more future-proof (and it does not add much overhead, it's done by Travis :) ). And if I test all the PHP versions, why not test the different Symfony versions as well? It's still a much lighter configuration than before, but test most of the cases.